src/planet-4-151612/openresty/bin/nginx_configure_cloudflare_cron.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+# Setup daily cron job to get CloudFlare updated ips
+# by running the nginx_configure_cloudflare.sh script with mods
+
+CLOUDFLARE_DAILY_CRON_FILE_PATH="/etc/cron.daily/nginx_update_cloudflare_ips"
```

Comment:
Perhaps test against a global variable, and enable disable the cron job according to `CLOUDFLARE_ENABLED` instead? this way the jobs only run when we need them.

```bash
if [ "${CLOUDFLARE_ENABLED,,}" != "true" ]
then
  chmod 644 "$CLOUDFLARE_DAILY_CRON_FILE"
else 
  chmod 755 "$CLOUDFLARE_DAILY_CRON_FILE"
fi
```

Another question, have you confirmed the scripts don't run when not executable?

Unit tests alongside new functionality would be preferable.

Reply:
- added global variable to real ip and cloudflare config file
- added unit tests for cloudflare and cron jobs